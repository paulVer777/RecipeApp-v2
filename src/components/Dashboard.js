import React from 'react'
import RecipeCreator from '../components/RecipeCreator'

const Dashboard = () => (
    <div className = 'dashboard'>
        <h2>Dasboard</h2>
        <RecipeCreator />
    </div>
)
    
export default Dashboard