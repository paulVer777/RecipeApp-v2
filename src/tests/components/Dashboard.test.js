import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Dashboard from '../../components/Dashboard';


test('Should render Dashboard correctly', () => {

    const renderer = new ReactShallowRenderer()
    renderer.render(<Dashboard />)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})



