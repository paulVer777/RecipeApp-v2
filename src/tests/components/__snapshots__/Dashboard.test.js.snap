// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Dashboard correctly 1`] = `
<div
  className="dashboard"
>
  <h2>
    Dashboard 
  </h2>
  <Connect(RecipeCreator) />
  <Connect(Filters) />
  <Connect(RecipeList) />
</div>
`;
